HelpSource/Classes/DrawGridX.schelp

```diff
@@ -0,0 +1,28 @@
+CLASS:: DrawGridX
+SUMMARY:: A subclass of DrawGrid, used for setting up its horizontal size
+CATEGORIES:: GUI>Accessories
+RELATED:: Classes/GridLines, Classes/Plotter, Classes/UserView, Classes/DrawGrid
+
+DESCRIPTION::
+DrawGridX, is a class instantiated by DrawGrid Class, in its init functon, to set the horizontal size of a GridUI Element, just as seen in the code below
+
+CLASSMETHODS::
+
+METHOD:: copy
+used for copying this instance
+
+EXAMPLES::
+
+CODE::
+// taken from Grid.sc class file
+init { arg bounds,h,v;
+		var w;
+		x = DrawGridX(h);
+		y = DrawGridY(v);
+		this.bounds = bounds;
+		this.font = Font( Font.defaultSansFace, 9 );
+		this.fontColor = Color.grey(0.3);
+		this.gridColors = [Color.grey(0.7),Color.grey(0.7)];
+	}
```

Comment:
@tiagmoraismorgado IMHO This class code example does not really fit in an help file